src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -0,0 +1,216 @@
+        ),
+    ],
+    tags=aws_config.tags,
+    vpc_id=mitxpro_vpc["id"],
+)
+
+mitxpro_edxapp_security_group = ec2.SecurityGroup(
+    f"mitxpro-edxapp-access-{env_suffix}",
+    name=f"mitxpro-edxapp-access-{env_suffix}",
+    description="Access control to mitxpro_edxapp",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            cidr_blocks=[mitxpro_vpc["cidr"]],
+            ipv6_cidr_blocks=[mitxpro_vpc["cidr_v6"]],
+            protocol="tcp",
+            from_port=22,
+            to_port=22,
+            description="mitxpro_vpc ssh access",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            cidr_blocks=[operations_vpc["cidr"]],
+            ipv6_cidr_blocks=[operations_vpc["cidr_v6"]],
+            protocol="tcp",
+            from_port=22,
+            to_port=22,
+            description="operations_vpc ssh access",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            cidr_blocks=["0.0.0.0/0"],
+            ipv6_cidr_blocks=["::/0"],
+            protocol="tcp",
+            from_port=80,
+            to_port=80,
+            description="HTTP access",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            cidr_blocks=["0.0.0.0/0"],
+            ipv6_cidr_blocks=["::/0"],
+            protocol="tcp",
+            from_port=443,
+            to_port=443,
+            description="HTTPS access",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            cidr_blocks=[mitxpro_vpc["cidr"]],
+            ipv6_cidr_blocks=[mitxpro_vpc["cidr_v6"]],
+            protocol="tcp",
+            from_port=18040,
+            to_port=18040,
+            description="Xqueue access",
+        ),
+    ],
+    tags=aws_config.tags,
+    vpc_id=mitxpro_vpc["id"],
+)
+
+mitxpro_edx_worker_security_group = ec2.SecurityGroup(
+    f"mitxpro-edx_worker-access-{env_suffix}",
+    name=f"mitxpro-edx_worker-access-{env_suffix}",
+    description="Access control to mitxpro_edx_worker",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[mitxpro_edxapp_security_group.id],
+            protocol="tcp",
+            from_port=18040,  # noqa: WPS432
+            to_port=18040,  # noqa: WPS432
+            description="Xqueue access",
+        )
+    ],
+    tags=aws_config.tags,
+    vpc_id=mitxpro_vpc["id"],
+)
+
+mitxpro_edxapp_db_security_group = ec2.SecurityGroup(
+    f"mitxpro-edxapp-db-access-{env_suffix}",
+    name=f"mitxpro-edxapp-db-access-{env_suffix}",
+    description="Access from the mitxpro VPC to the mitxpro edxapp database",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[
+                mitxpro_edxapp_security_group.id,
+                mitxpro_edx_worker_security_group.id,
+                consul_server_security_group.id,
+            ],
+            protocol="tcp",
+            from_port=3306,  # noqa: WPS432
+            to_port=3306,  # noqa: WPS432
+        )
+    ],
+    tags=aws_config.tags,
```

Comment:
We should add a `Name` tag as well using the `merged_tags` method.

`tags=aws_config.merged_tags({'Name': 'database-name-tag-value'})`

Reply:
What's the purpose of this?

Reply:
It adds the `Name` tag to the already defined tags so that we can add arbitrary tags in addition to the baseline values and return the merged dictionary.